wptrunner/wptrunner.py

```diff
@@ -148,6 +148,9 @@ def run_tests(config, test_paths, product, **kwargs):
 
         kwargs["pause_after_test"] = get_pause_after_test(test_loader, **kwargs)
 
+        env_options["host"] = kwargs.get("host") or env_options["host"]
+        env_options["external_host"] = kwargs.get("external_host") or env_options["external_host"]
```

Comment:
Actually, this doesn't work, because when the options are parsed, `kwargs["host"]` is always set, it is just set to `None` if the `--host` is not present. So I think the original code was doing the right thing.